### PR TITLE
View/interview planning

### PR DESCRIPTION
**Pull request by LeoCode0**
## Checklist ✅

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [ ] Tests are included
- [ ] Documentation is changed or added
- [ ] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

resolves [Issue ID]

### CODE REVIEW

File: src/components/InterviewInformation/styles.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    maring-botton: 15px;
```
Comment (odagora) [inline src/components/InterviewInformation/styles.js:7]: margin-bottom

File: src/components/FormPlanning/index.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                <TextField
+                    id="outlined-basic"
+                    label="Url or phisical direction"
+                    variant="outlined"
+                    sx={{ width: '100%' }}
+                />
+            </Box>
+            <Box sx={{ mb: 2 }}>
+                <FormControl sx={{ width: '100%' }}>
+                    <InputLabel id="demo-simple-select-label">Who is going to interview?</InputLabel>
```
Comment (odagora) [inline src/components/FormPlanning/index.jsx:85]: The text is crossing with input form

File: src/components/FormPlanning/index.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+
+import { Box, TextField, Select, InputLabel, MenuItem, FormControl, Button } from '@mui/material';
```
Comment (ZajithCorro) [inline src/components/FormPlanning/index.jsx:4]: This something we have to talk like grup. Import from `@mui/material` or `@mui/material/Box`

File: src/components/FormPlanning/index.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+import TimePicker from '@mui/lab/TimePicker';
+
+export const FormPlanning = () => {
+    const [date, setDate] = useState(null);
+    const [hour, setHour] = useState(null);
+    const [type, setType] = useState(null);
+    const [platform, setPlatform] = useState(null);
```
Comment (ZajithCorro) [inline src/components/FormPlanning/index.jsx:14]: Maybe you can implement React Hook Form 🤔

File: src/components/FormPlanning/index.jsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+                <Select
+                    labelId="demo-simple-select-label"
+                    id="demo-simple-select"
+                    value={platform}
+                    label="Interview type"
+                    onChange={changePlatform}
+                >
+                    <MenuItem value="zoom">Zoom</MenuItem>
+                    <MenuItem value="google-meet">Google meet</MenuItem>
+                    <MenuItem value="microsoft-teams">Microsoft teams</MenuItem>
```
Comment (ZajithCorro) [inline src/components/FormPlanning/index.jsx:72]: This select is not useful if the recruiter doesn't select the remote interview type

File: src/components/FormPlanning/index.jsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+                    <InputLabel id="demo-simple-select-label">Interview type</InputLabel>
+                    <Select
+                        labelId="demo-simple-select-label"
+                        id="demo-simple-select"
+                        value={type}
+                        label="Interview type"
+                        onChange={handleChange}
+                    >
+                        <MenuItem value="remote">Remote</MenuItem>
+                        <MenuItem value="presencial">Presencial</MenuItem>
```
Comment (ZajithCorro) [inline src/components/FormPlanning/index.jsx:57]: Sounds better **Face to Face**. What do you think @odagora ? 🤔

File: src/components/InterviewInformation/index.jsx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+                    </Typography>
+                </Grid>
+                <Grid item xs={4}>
+                    <Typography variant="subtitle2" align="center">
+                        Platform {platform}
+                    </Typography>
+                </Grid>
+            </Grid>
+            <Typography variant="body1" sx={{ mb: 1, mt: 1 }}>
+                Location: <a href={location}>{location}</a>
```
Comment (ZajithCorro) [inline src/components/InterviewInformation/index.jsx:84]: Location sounds more like a physical place instead just the link

File: src/routes/index.jsx [hunk 8]
```diff
@@ -4,6 +4,7 @@ import { HashRouter, Routes as Switch, Route } from 'react-router-dom';
 import { NotFound } from 'Pages/NotFound';
 import { CreateVacancies } from 'Pages/CreateVacancies';
 import { EditVacancies } from 'Pages/EditVacancies';
+import { InterviewPlanning } from '../pages/InterviewPlanning';
```
Comment (ZajithCorro) [inline src/routes/index.jsx:7]: The alias. Instead of `../pages/InterviewPlanning` just `Pages/InterviewPlanning`

**Review by ZajithCorro — COMMENTED** [review]
We can discuss this the slack channel

File: src/components/FormPlanning/index.jsx (hunk 3 above)
Comment (LeoCode0) [reply src/components/FormPlanning/index.jsx:4]: Yeah, I think that we can talk this in the next meet

File: src/components/FormPlanning/index.jsx (hunk 6 above)
Comment (LeoCode0) [reply src/components/FormPlanning/index.jsx]: Fixed!

File: src/components/InterviewInformation/styles.js (hunk 1 above)
Comment (LeoCode0) [reply src/components/InterviewInformation/styles.js]: Fixed! A little typo haha

File: src/routes/index.jsx (hunk 8 above)
Comment (LeoCode0) [reply src/routes/index.jsx]: Fixed!

**Review by LeoCode0 — COMMENTED** [review]
I added and fixed the following details:
- Implement React hook form
- Fix typo in styles
- Change some words to improve the UX
-

**Review by LeoCode0 — COMMENTED** [review]
The last change was delete the card buttons as all recommend in the past meeting

**Review by odagora — APPROVED** [review]

**Pull request merged**